specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1335,23 +1799,27 @@
           "description": "Hours of sun."
         },
         "degreeDaySummary": {
+          "x-ms-client-name": "meanTemperatureDeviation",
           "$ref": "#/definitions/DegreeDaySummary",
-          "description": "Summary of Heating Degree Day or Cooling Degree Day information"
+          "description": "Summary for mean temperature of Heating Degree Day or Cooling Degree Day information"
         },
         "airAndPollen": {
+          "x-ms-client-name": "airQuality",
           "type": "array",
           "items": {
             "$ref": "#/definitions/AirAndPollen"
           },
-          "description": "airAndPollen"
+          "description": "air quality"
```

Comment:
Should at least capitalize the description to be consistent with sibling properties.